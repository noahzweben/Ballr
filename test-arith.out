  ./test-arith failed on ./test-arith
