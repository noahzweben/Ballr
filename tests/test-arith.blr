gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {
		int val1 = 1 + 2 * 3 - 1;
		print(val1);
		int val2 = 1 + 10/2 + 5;
		print(val2);  
		int neg_val = -1 + 2 * 3 - 1;
		print(neg_val);
	}
}
