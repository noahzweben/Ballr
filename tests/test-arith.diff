1c1,5
<   ./test-arith failed on ./test-arith
---
> 6
> 11
> 4
> SDL could not initialize! SDL Error: No available video device
> Window initialization failed.
