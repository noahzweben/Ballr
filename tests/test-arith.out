6
11
4
SDL could not initialize! SDL Error: No available video device
Window initialization failed.
